sdk/network/mgmt/src/main/java/com/azure/management/network/implementation/ExpressRouteCircuitPeeringImpl.java

```diff
@@ -10,21 +10,28 @@
 import com.azure.management.network.Ipv6ExpressRouteCircuitPeeringConfig;
 import com.azure.management.network.models.ExpressRouteCircuitPeeringInner;
 import com.azure.management.network.models.ExpressRouteCircuitPeeringsInner;
+import com.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.azure.management.resources.fluentcore.model.Refreshable;
 import com.azure.management.resources.fluentcore.model.implementation.CreatableUpdatableImpl;
 import com.azure.management.resources.fluentcore.utils.Utils;
 import java.util.Arrays;
 import reactor.core.publisher.Mono;
 
-class ExpressRouteCircuitPeeringImpl
+interface ExpressRouteCircuitPeeringParent<ModelT, InnerT>
+    extends GroupableResource<NetworkManager, InnerT>,
+    Refreshable<ModelT> {}
```

Comment:
Could be reviewed carefully.

Reply:
Any detail about this?

Reply:
In order to deal with the peering inner in route filter. https://github.com/Azure/azure-sdk-for-java/pull/10421/files#diff-54b158e5bec26f0a0848f449f42c704cR55

Maybe I think I could remove the interface and use in ExpressRouteCircuitPeering template.

Reply:
I have removed it and add template in ExpressRouteCircuitPeeringImpl